src/register-org/components/document-exchange-reference/document-exchange-reference.component.html

```diff
@@ -5,41 +5,37 @@
       <xuilib-hmcts-error-summary [errorMessages]="dxErrors" [header]="'There is a problem'">
       </xuilib-hmcts-error-summary>
       <span class="govuk-caption-xl">Register an organisation</span>
+      <h1 class="govuk-heading-xl">
```

Comment:
@johnbenjamin-hmcts - please revert all changes here and reinstate the `<fieldset>` structure. Radio buttons should use it - see https://design-system.service.gov.uk/components/radios/

Reply:
@Daniel-Lam - fieldset is used for grouping controls together, whereas in this case it is only Yes No.

https://design-system.service.gov.uk/components/fieldset/

Reply:
@Daniel-Lam - I have updated the code now.